src/components/StorageOverview/DataResiliency/DataResiliency.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, UtilizationBar } from 'patternfly-react';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { getDataResiliencyData } from '../../../selectors/prometheus/selectors';
+
+const DataResiliencyStatusBody = ({ status }) =>
+  status ? (
+    <React.Fragment>
+      <div className="kubevirt-data-resiliency__status-title--ok">Your Data Is Resilient</div>
+      <div className="kubevirt-data-resiliency__icon--ok">
+        <Icon type="fa" name="check-circle" size="3x" />
```

Comment:
is it possible to set the className just on the icon instead?

Reply:
I tried but did not get desired changes!